lispy-test.el

```diff
@@ -1880,12 +1880,12 @@ Insert KEY if there's no command."
                    "#?@(|)")))
 
 (ert-deftest lispy-braces ()
-  (should (string= (lispy-with "\"a regex \\\\|\"" "{")
+  (should (string= (lispy-with-clojure "\"a regex \\\\|\"" "{")
```

Comment:
This should be kept as `lispy-with`, this is a valid test for dealing with Elisp regexes.